src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -261,6 +261,15 @@
   <data name="SetViewStateUserKeyMessage" xml:space="preserve">
     <value>The subclass {0} of Page does not set the ViewStateUserKey property in method OnInit</value>
   </data>
+  <data name="UseContainerLevelAccessPolicy" xml:space="preserve">
+    <value>Use Container Level Access Policy</value>
+  </data>
+  <data name="UseContainerLevelAccessPolicyDescription" xml:space="preserve">
+    <value>No access policy identifier is used which makes it a non-revocable token limiting the SAS access to only the request customer's id. To grant longer-term permissions or to retain the ability to modify or revoke permissions after handing URL out, it's better to use container-level access policies.</value>
```

Comment:
>No access policy identifier is used which makes it a non-revocable token limiting the SAS access to only the request customer's id [](start = 11, length = 130)

Let's just say "No access policy identifier is specified, making tokens non-revocable."